### PR TITLE
Kevin/user login endpoint

### PR DESCRIPTION
**Pull request by KevinetPeng**

### CODE REVIEW

File: backend/Controller.js [hunk 1]
```diff
@@ -28,3 +31,64 @@ exports.getUser = async (req, res) => {
...
+                })
+        },
+        //access token refresh error
+        (err) => {
+            res.json("Unable to refresh access token.");
+            console.log(err);
+        }
+    );
+
+
```
Comment (brendanlai) [inline backend/Controller.js]: extra empty line (delete for formatting??)

**Review by Brendanlai — APPROVED** [review]
Looks good to me

**Review by Remyzh — COMMENTED** [review]
I was wondering what happened to the "const { Mongoose } = require('mongoose');" line that I have in my controller.js. Is it not needed anymore?

**Review by Remyzh — APPROVED** [review]
Looks good!

File: backend/Controller.js (hunk 1 above)
Comment (KevinetPeng) [reply backend/Controller.js]: Fixed!

**Pull request merged**